NextExample/pages/noreanimated.js

```diff
@@ -0,0 +1,30 @@
+import { StyleSheet, View, Text } from 'react-native';
+
+export default function Reanimated() {
```

Comment:
Or `App` or something else, `Reanimated` is weird here.
```suggestion
export default function Page() {
```